run.rb

```diff
@@ -71,43 +71,47 @@ def cli_prompt_for_config_values()
 markdown_content = "Data fetched on #{Time.now.strftime("%Y-%m-%d %H:%M")}\n\n"
 subtasks = []
 
+tasks_no_subtasks = tasks.map do |task|
+  task[:subtasks] = []
+  task
+end
```

Comment:
@jamespeilunli we have to create a duplicate array in order to safely remove elements from it while iterating over the original

Reply:
I add the subtasks attribute here instead of below (formerly line 87), otherwise it was getting wiped out every time and only the _last_ subtask made it onto the subtask list.